packages/tools/devtools/devtools-core/README.md

```diff
@@ -1,12 +1,8 @@
-# @fluidframework/devtools-core
```

Comment:
@chentong7 it looks like a handful of the package READMEs had their contents replaced with the contents of the `cell` package's README. We'll need to revert these changes.

Reply:
Looks like the affected packages include:
* counter
* devtools
* devtools-core
* fluid-framework